hackerdrinks/content/events/20220519-timbre-plus.md

```diff
@@ -0,0 +1,10 @@
++++
+title = "RubySG x Hackerdrinks - Timbre+"
+date = "19 May 2022"
+day = "Thursday"
+time = "7.00pm"
```

Comment:
the plan is 8pm RubySG, 9pm hackerdrinks

Reply:
Our guys will usually come at 7pm haha, just join us after!